index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../css/styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <img src="../src/assets/images/header-shapes-mobile@2x.svg" alt="background" class="bg_shape">
+        <nav>
+            <ul class="flex-nav">
+                <li class="logo">
+                    <a href="#"><img alt="logo" src="../src/assets/images/my_logo.png"></a>
+                </li>
+                <li class="menu">
+                    <a href=""><img alt="Menu" src="../src/assets/images/img_icon.png"></a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section class="wrap">
+        <h1>I am TEEJAY! Happy to have you here today!!</h1>
+            <p>I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+                <a href="#">LET'S CONNECT</a>
+            <ul class="social">
+                <li><a href=""><img src="../src/assets/images/Vector.svg" alt="Facebook" width="24" height="20"></a></li>
+                <li><a href=""><img src="../src/assets/images/Linkedin.svg" alt="Linkedin" width="24" height="20"></a></li>
+                <li><a href=""><img src="../src/assets/images/Vector_1.svg" alt="Instagram" width="24" height="20"></a></li>
```

Comment:
- [ ] Please as suggested by the previous reviewer try to adjust your design like padding and margins to match with Figma design and that will make your project looks good.you got this 👍 

Your design:
![image](https://user-images.githubusercontent.com/90524466/185648319-85531241-e7f1-4169-bb04-1083830bbcdd.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185648987-7bfc2862-eea6-4397-a9be-5f4ac06244e7.png)

Reply:
![image](https://user-images.githubusercontent.com/5064641/185657451-bd941465-2814-495e-96d7-593d880c79d4.png)

Reply:
@mugishasam123 Ok, have made the changes.
This is my last chance to request a review so hope it goes through successfully